packages/manager/modules/pci/src/projects/project/public-gateways/onboarding/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,12 @@
+{
+  "pci_projects_project_public_gateways_onboarding": "Public Gateway",
+  "pci_projects_project_public_gateways_onboarding_content1": "Une instance entièrement dédiée installée dans un réseau privé peut nécessiter l’accès à des ressources internet. Public Gateway est un moyen de connexion sécurisé, sans exposition directe de l’instance à Internet.",
+  "pci_projects_project_public_gateways_onboarding_content2": "Une application web exposée à Internet doit posséder une adresse IP. Pour cela, une IP flottante constitue la meilleure solution. Il s'agit d'une IP flexible et agnostique, pouvant être déplacée entre des services Public Cloud. Pour une telle utilisation, elle doit être configurée sur une Public Gateway et directement liée à une instance privée ou au Load Balancer. Et ce, pour une meilleure évolutivité du service.",
+  "pci_projects_project_public_gateways_onboarding_action_label": "Créer une Public Gateway",
+  "pci_projects_project_public_gateways_onboarding_guides_creat-private-network-with-gateway_title": "Créer un réseau privé avec Public gateway depuis l'espace client OVHcloud",
+  "pci_projects_project_public_gateways_onboarding_guides_creat-private-network-with-gateway_description": "",
```

Comment:
Empty ? We need it ?

Reply:
Removed as we don't have any description as of now, if it comes will add it again.

Reply:
Not issue if you know you will complete it before we prod.